docker-image/Dockerfile.ppc64le

```diff
@@ -1,22 +1,24 @@
 ARG QEMU_IMAGE=calico/go-build:latest
 FROM ${QEMU_IMAGE} as qemu
 
-FROM ppc64le/alpine:3.8 as base
+FROM ppc64le/debian:9.8-slim as base
 MAINTAINER Shaun Crampton <shaun@tigera.io>
 
 # Enable non-native builds of this image on an amd64 hosts.
 # This must before any RUN command in this image!
-# we only need this for the intermediate "base" image, so we can run all the apk and other commands
+# we only need this for the intermediate "base" image, so we can run all the apt-get and other commands
 # and this is only because of using older kernels
 # when running on a kernel >= 4.8, this will become less relevant
 COPY --from=qemu /usr/bin/qemu-ppc64le-static /usr/bin/
 
 # Since our binary isn't designed to run as PID 1, run it via the tini init daemon.
-RUN apk add --update tini
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-static-ppc64el /sbin/tini
```

Comment:
Kind of odd that the Tini binary is called `ppc64el` and our arch is called `ppc64le`. Pretty sure they are the same.